orderByRelation previewFeatures gives an error

I have tried to use  the orderByRelation previewFeatures released in version "@prisma/client": "2.19.0",

```
generator client {
  provider = "prisma-client-js"
  previewFeatures=["orderByRelation"]
}
```

After generating the client ` npx prisma generate` and trying to start the dev server i've got the below issue :

```
Error: Could not find ordering argument for Blog.posts
    at SchemaBuilder.argsFromQueryOrModelField (C:\Users\v_2190\nexus-plugin-prisma\examples\blog\node_modules\nexus-plugin-prisma\src\builder.ts:663:15)
    at SchemaBuilder.determineArgs (C:\Users\v_2190\nexus-plugin-prisma\examples\blog\node_modules\nexus-plugin-prisma\src\builder.ts:602:19)
    at SchemaBuilder.buildArgsFromField (C:\Users\v_2190\nexus-plugin-prisma\examples\blog\node_modules\nexus-plugin-prisma\src\builder.ts:584:17)
    at SchemaBuilder.buildFieldConfig (C:\Users\v_2190\nexus-plugin-prisma\examples\blog\node_modules\nexus-plugin-prisma\src\builder.ts:574:18)
    at Proxy.fieldPublisher (C:\Users\v_2190\nexus-plugin-prisma\examples\blog\node_modules\nexus-plugin-prisma\src\builder.ts:525:34)
    at Object.definition (C:\Users\v_2190\nexus-plugin-prisma\examples\blog\src\schema\types\Blog.ts:10:13)
    at SchemaBuilder.walkOutputType (C:\Users\v_2190\nexus-plugin-prisma\examples\blog\node_modules\nexus\src\builder.ts:1532:9)
    at SchemaBuilder.walkTypes (C:\Users\v_2190\nexus-plugin-prisma\examples\blog\node_modules\nexus\src\builder.ts:634:16)
    at SchemaBuilder.getFinalTypeMap (C:\Users\v_2190\nexus-plugin-prisma\examples\blog\node_modules\nexus\src\builder.ts:804:10)
    at makeSchemaInternal (C:\Users\v_2190\nexus-plugin-prisma\examples\blog\node_modules\nexus\src\builder.ts:1662:92)
[ERROR] 10:25:33 Error: Could not find ordering argument for Blog.posts
```
Many thanks in advance

--- a/src/builder.ts
+++ b/src/builder.ts
@@ -655,9 +655,13 @@ export class SchemaBuilder {
 
     if (publisherConfig.ordering) {
       const orderByTypeName = `${field.outputType.type}OrderByInput`
+      const orderByTypeNamePreviewFeature = `${field.outputType.type}OrderByWithRelationInput`
       const orderByArg = field.args.find(
-        (arg) => arg.inputType.type === orderByTypeName && arg.name === 'orderBy'
-      )
+        (arg) =>
+          (arg.inputType.type === orderByTypeName ||
+            arg.inputType.type === orderByTypeNamePreviewFeature) &&
+          arg.name === 'orderBy'
+      );
 
       if (!orderByArg) {
         throw new Error(`Could not find ordering argument for ${typeName}.${field.name}`)
@@ -665,7 +669,7 @@ export class SchemaBuilder {
 
       const inputType = this.handleInputObjectCustomization({
         fieldWhitelist: publisherConfig.ordering,
-        inputTypeName: orderByTypeName,
+        inputTypeName: orderByArg.inputType.type,
         fieldName: field.name,
         graphQLTypeName: typeName,
         isWhereType: false,

